lib/FieldType/FieldTypeFormMapperDispatcher.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\FieldType;
+
+use EzSystems\RepositoryForms\Data\Content\FieldData;
+use EzSystems\RepositoryForms\Data\FieldDefinitionData;
+use Symfony\Component\Form\FormInterface;
+use InvalidArgumentException;
+
+/**
+ * Registry for FieldType form mappers.
+ */
+class FieldTypeFormMapperDispatcher implements FieldTypeFormMapperDispatcherInterface
+{
+    /**
+     * FieldType form mappers, indexed by FieldType identifier.
+     *
+     * @var FieldTypeFormMapperInterface[]
+     */
+    private $mappers = [];
+
+    public function getMappers()
+    {
+        return $this->mappers;
+    }
+
+    public function addMapper(FieldFormMapperInterface $mapper, $fieldTypeIdentifier)
+    {
+        if ($mapper instanceof FieldTypeFormMapperInterface) {
+            @trigger_error(
+                'The FieldTypeFormMapperInterface interface is deprecated in ezsystems/repository-forms 1.1, ' .
+                "and will be removed in version 2.0\n" .
+                'Use FieldValueFormMapperInterface and FieldDefinitionFormMapperInterface instead',
+                E_USER_DEPRECATED
+            );
+        } elseif (!$mapper instanceof FieldValueFormMapperInterface && !$mapper instanceof FieldDefinitionFormMapperInterface) {
+            throw new \InvalidArgumentException('Expecting a FieldValueFormMapperInterface or FieldDefinitionFormMapperInterface');
+        }
+
+        $this->mappers[$fieldTypeIdentifier] = $mapper;
+    }
+
+    public function addMappers(array $mappersArray)
```

Comment:
Added that yesterday, it's not even used yet. Having this method allows us to add one big method call in the compiler pass, with an array of N mappers, instead of N method calls of 1 mapper. 

I think i'll add it to the interface. Even though somehow, I'd like this better if the `addMapper(s)` methods were in another interface than `map`.